### PR TITLE
fix(*): close connections for initDb and dropDb

### PR DESCRIPTION
**Pull request by mebibou**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -149,16 +149,17 @@ function initDb(url, data) {
...
 function dropDb(url) {
   return client.connect(url)
-    .then(db => db.collections())
-    .then(collections => {
-      const requests = collections.map(col => col.drop())
-      return Promise.all(requests)
+    .then(db => {
+      return db.collections().then(collections => {
+        const requests = collections.map(col => col.drop())
+        return Promise.all(requests).then(() => db.close())
```
Comment (mikhail-angelov) [inline index.js]: this line in not in right place

Comment (mebibou) [reply index.js]: I'm not sure I understand, it is aligned with the previous line

Comment (mikhail-angelov) [reply index.js]: maybe like this
```
function dropDb(url) {
  return client.connect(url)
    .then(db => {
      return db.collections()
          .then(collections => {
              const requests = collections.map(col => col.drop())
              return Promise.all(requests)
          })
         .then(() => db.close())
    })
}
```

Comment (mebibou) [reply index.js]: lol same thing but sure

**Pull request merged**